feat: Redoing home and moving getting started to actors/running and actors/development sections (#542)
apify/apify-docs: sources/platform/actors/development/index.md

> Crawlee is a web scraping and browser automation library. <br />
> It helps you build reliable crawlers. Fast.

Let's run it! To run your Actor, you need to build it first, so let's press the "Build" button below the source code.

![Actor source code](./images/home/actor-source-code.png)

### 3. Build it

After you press the "Build" button, the UI transitions to the "Build" tab, and you can see the progress of the build, including the Docker build log.

![Actor build](./images/home/actor-build.png)

Notice that the UI includes four tabs - (1) Code, (2) Build, (3) Input, and (4) Run. This represents the Actor creation flow where you need first to take the source code and build it. Once you have a build, you give it input which results in an Actor run.

### 4. Run it

Once the Actor is built, you can look at its input, which consists of one field - Start URL, the URL where crawling starts. Below the input, you can adjust the run options consisting of build, timeout, and memory limit.

![Actor input](./images/home/actor-input.png)

> After pressing a run button, the actor run gets created, and you can view its log and other details. Results will shortly start popping up in the output tab. You can always stop the run using the "Abort" button.

Reviewer: ```suggestion
You can start an Actor run by pressing the **Start** button at the bottom of the page. Once the run is created, you can view its log and other details, including the **Results** which will shortly start popping up in the output tab. You can always stop the run using the **Abort** button.
```

![Actor run](./images/home/actor-run.png)

### 5. Iterate

> Now you can get back to the source code and modify the actor to match your needs. To learn about all the Apify Platform features and best practices, continue to the following chapter of this section or visit [Apify Academy](/academy).

Reviewer: ```suggestion
Now you can get back to the source code and modify the actor to match your needs. To learn about all the Apify Platform features and best practices, continue to the following chapter of this section or visit the [Apify Academy](/academy).
```

## Build Actor locally

> Prerequisite - You need to have [Node.js](https://nodejs.org/en/) version 16 or higher with NPM installed on your computer.

### 1. Install Apify CLI

On macOS (or Linux), you can install the Apify CLI via the Homebrew package manager.

```bash
brew install apify/tap/apify-cli
```

Otherwise, use [NPM](https://www.npmjs.com/) to install the Apify CLI.

```bash
npm -g install apify-cli
```

For more installation and advanced usage information, see the [Apify CLI documentation](https://docs.apify.com/cli/).

### 2. Create your Actor

> Use the

Reviewer: ```suggestion
You can use the following command to create an Actor:
```

```bash
apify create
```

> Command to create your actor. We choose `my-actor` as a name, JavaScript as the language, and "PuppeteerCrawler in JavaScript" as a template. CLI will then:

Reviewer: ```suggestion
You will be prompted to name your Actor (e.g, my-actor), and choose the programming language you would like to use (JavaScript, TypeScript, or Python). Next, you will be presented with a list of Actor development templates. After choosing your preferred template the CLI will:
```

- Create a `my-actor` directory containing boilerplate code.

> - Install all the NPM dependencies.

Reviewer: ```suggestion
- Install all the project dependencies.
```

![Creation](./images/home/actor-create.gif)

> Now switch to the actor directory:

Reviewer: ```suggestion
Finally, switch to your newly created Actor directory:
```

```
cd my-actor
```

### 3. Explore the source code

> Now it's time to explore the source code. The important parts are:

Reviewer: ```suggestion
Next, let's explore the Actor's source code. We will use the "PuppeteerCrawler in JavaScript" template code as an example, but all Actor templates follow a similar organizational pattern. The important parts are:
```

Next, let's explore the Actor's source code. We will use the "PuppeteerCrawler in JavaScript" template code as an example, but all Actor templates follow a similar organizational pattern. The important parts are:

- `src` directory and, namely, the `src/main.js` file, which is the actual code of an actor.
- `actor` directory containing actor's definition files such as `actor.json` and `Dockerfile`. These are important once you deploy your code to the Apify Platform.
- `storage` directory containing the local emulation of [Apify Storage](../storage), namely [key-value store](../storage/key-value-store), [dataset](../storage/dataset), and [request queue](../storage/request-queue).

![Actor source code](./images/home/actor-local-code.png)

> The following chapters will teach you more about an actor's structure and files.

### 4. Run it locally

To run the actor, call:

```bash
apify run
```

Shortly after that, you will see the log of the actor. The results will be stored in the local dataset under the `storage/dataset/default` directory.

![Actor source code](./images/home/actor-local-run.png)

### 5. Deploy it to Apify Platform

> In order to push your actor to Apify Platform, you need to first sign in to Apify with the CLI tool:

Reviewer 2: ```suggestion
In order to push your Actor to the Apify platform, you need to first sign in to Apify with the CLI tool:
```

```bash
apify login
```

> And then, you can push your actor under your Apify account with the following:

Reviewer 2: ```suggestion
And then, you can push your Actor under your Apify account with the following:
```
